tests/std/tests/P0896R4_ranges_iterator_machinery/test.cpp

```diff
@@ -2770,6 +2772,26 @@ namespace iter_ops {
     }
 } // namespace iter_ops
 
+namespace insert_iterators {
+    template <class Container>
+    constexpr bool test() {
+        using std::back_insert_iterator, std::front_insert_iterator, std::insert_iterator;
+        using std::default_initializable, std::iter_difference_t, std::ptrdiff_t, std::same_as;
+
+        STATIC_ASSERT(default_initializable<back_insert_iterator<Container>>);

```

Comment:
I believe we should also check `std::is_nothrow_default_constructible` given that these are explicitely `noexcept`